eng/pipelines/templates/jobs/native.live.tests.yml

```diff
@@ -124,6 +124,8 @@ jobs:
               SubscriptionConfiguration: ${{ parameters.CloudConfig.SubscriptionConfiguration }}
               SubscriptionConfigurations: ${{ parameters.CloudConfig.SubscriptionConfigurations }}
               EnvVars: ${{ parameters.EnvVars }}
+              ${{ if eq('true', parameters.UseFederatedAuth) }}:
```

Comment:
I don't see this in the list of parameters for this template.

Reply:
Good catch. This introduced new errors: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3913001&view=results